crates/macros/src/lib.rs

```diff
@@ -165,17 +165,17 @@ pub fn connect(attr: proc_macro::TokenStream) -> proc_macro::TokenStream {
         use futuresdr::runtime::Block;
         use futuresdr::runtime::Flowgraph;
 
-        struct Foo;
+        struct FgOp;
```

Comment:
I hope you're ok with this change. When I first saw `Foo` in an expanded macro I thought maybe something inadvertant leaked in. :D